moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectTypes.js

```diff
@@ -0,0 +1,190 @@
+import React, { useState } from "react";
+import {
+  Box,
+  Checkbox,
+  Grid,
+  Icon,
+  Input,
+  ListItemText,
+  MenuItem,
+  Select,
+  Typography,
+} from "@material-ui/core";
+
+import ProjectSummaryLabel from "./ProjectSummaryLabel";
+import { useMutation } from "@apollo/client";
+import { PROJECT_UPDATE_TYPES } from "../../../../queries/project";
+
+/**
+ * ProjectSummaryProjectTypes Component
+ * @param {Number} projectId - The id of the current project being viewed
+ * @param {Object} data - The data object from the GraphQL query
+ * @param {function} refetch - The refetch function from apollo
+ * @param {Object} classes - The shared style settings
+ * @param {function} snackbarHandle - The function to show the snackbar
+ * @returns {JSX.Element}
+ * @constructor
+ */
+const ProjectSummaryProjectTypes = ({
+  projectId,
+  data,
+  refetch,
+  classes,
+  snackbarHandle,
+}) => {
+  /**
+   * Helper initial state lists
+   */
+  const typeList = data?.moped_types ?? [];
+  const typeDict = typeList.reduce(
+    (prev, curr) => ({
+      ...prev,
+      ...{ [curr.type_id]: curr.type_name },
+    }),
+    {}
+  );
```

Comment:
[Looking back](https://github.com/cityofaustin/atd-moped/pull/450/files#diff-b26f3e28654dd086f13a0473547d1ae187d5222c3156708a18623633ca34a270R39-R45) I wonder if I should have used a `.forEach` and something like `typeList[n.type_id] = n.type_name`.  It looks great, though! 👍  It works as expected.